addon/package.json

```diff
@@ -34,7 +34,7 @@
     "babel-eslint": "4.1.8",
     "eslint": "^1.10.3",
     "eslint-config-airbnb": "^0.0.8",
-    "jpm": "1.0.6",
+    "jpm": "1.1.3",
```

Comment:
When I ran these tests on a fresh CircleCI cache, there were issues with the add-on output filename. So, I just went ahead and upgraded jpm and dealt with it